src/openforms/templates/includes/page-header.html

```diff
@@ -17,8 +17,7 @@
         class="utrecht-link utrecht-link--openforms utrecht-page-header__openforms-return-url"
         href="{% firstof config.main_website '#' %}"
         title="{{ logo_alt }}"
-        aria-label="{{ logo_alt }}"
-    >{% if config.main_website %}{% trans "Back to website" %}{% endif %}</a>
+    >{% if config.main_website %}{{ logo_alt }}{% endif %}</a>
```

Comment:
what happens now if the organization name is not specified? This will lead to a broken sentence, no?

> Back to website of

In those situations I'd expect the string "Back to website" to be used.

Reply:
I've changed the `logo_alt` like you suggested. So when no organization name is specified, the `logo_alt` will return "Back to website"